pep-0728.rst

```diff
@@ -0,0 +1,322 @@
+PEP: 728
+Title: Inner Fields Annotations
```

Comment:
Could you be a bit more specific with this title? Before reading the PEP I had no idea what "Inner Fields Annotations" meant.